starter_code/app.js

```diff
@@ -15,4 +16,38 @@ app.get('/', (req, res, next) => {
 });
 
 
+app.get('/beers', (req, res, next) => {
+   let data = {
```

Comment:
I don't see the reason to use data thing here. You could've skipped this step completely.